v2/openapi/fb-provider-crosstransfer-api.yaml

```diff
@@ -0,0 +1,304 @@
+    CreateOnRampRoutingRequest:
+      type: object
+      required:
+        - idempotencyKey
+        - customerId
+        - source
+        - destination
+      properties:
+        idempotencyKey:
+          type: string
+          example: 76a33ed9-8aa2-4781-93f8-d2369c0c2b14
+        customerId:
+          type: string
+          example: 76a33ed9-8aa2-4781-93f8-d2369c0c2b14
+        source:
+          $ref: '#/components/schemas/OnRampSourceParameters'
+        destination:
+          $ref: '#/components/schemas/OnRampDestination'
+    OnRampDestination:
+      type: object
+      required:
+        - chain
+        - currency
+        - address
+      properties:
+        chain:
+          $ref: '#/x-schemas/Blockchain'
+        currency:
+          $ref: '#/x-schemas/NativeCryptocurrency'
+        address:
+          description: Address of the destination account.
+          type: string
+        memo:
+          description: Memo field for the destination address. Required for some blockchains.
+          type: string
+    OnRampSourceParameters:
+      type: object
+      required:
+        - currency
+      properties:
+        currency:
+          $ref: '#/x-schemas/NativeCryptocurrency'
+        developerFee:
+          $ref: '#/x-schemas/PositiveAmount'
+    OffRampSourceParameters:
+      type: object
+      required: [ chain, currency ]
+      properties:
+        chain:
+          $ref: '#/x-schemas/Blockchain'
+        currency:
+          $ref: '#/x-schemas/NativeCryptocurrency'
+        developerFee:
+          $ref: '#/x-schemas/PositiveAmount'
+        returnAddress:
+          type: string
+          description: >-
+            Blockchain address to return the funds in case of a failed transaction. 
+            MUST be on same blockchain network that is selected in the chain field.
+            Field will be ignored in case of Stellar chain.
+    OffRampDestination:
+      type: object
+      required: [ rail, currency, account ]
+      properties:
+        rail:
+          $ref: '#/x-schemas/FiatRail'
+        currency:
+          $ref: '#/x-schemas/NationalCurrency'
+        message:
+          type: string
+          description: >-
+            A message to be sent with a wire transfer. 
+            It can have at most 3 lines (separated by "\n"),
+            each line with a max length of 35 chars
+          maxLength: 109
+        reference:
+          type: string
+          description: >-
+            If rail is SEPA, this is the SEPA reference and must be from 6 to 140 characters. 
+            The allowed characters are a-z, A-Z, 0-9, spaces, ampersand (&), hyphen (-), 
+            full stop (.), and solidus (/). If rail is ACH, this is the ACH reference and
+            can be at most 10 characters, A-Z, a-z, 0-9, and spaces.
+        account:
+          oneOf:
+            - $ref: '#/x-schemas/ExternalAccountRef'
+            - $ref: '#/x-schemas/AccountRef'
```

Comment:
Fiat withdrawals are already defined in `FiatTransfer` object and it is practically the same.